backend/game_server/lib/cell.ex

```diff
@@ -0,0 +1,32 @@
+defmodule BernacleServer.Cell do
+	alias BernacleServer.{Entity,Cell}
+	alias BernacleServer.Helpers.{Physic,Vector}
+
+
+	def born(position, velocity) do
+		Entity.new
+			|> Entity.set_attribute(:position, position)
+			|> Entity.set_attribute(:velocity, velocity)
+			|> Entity.set_attribute(:mass, 5)
+			|> Entity.give_ability(:move, Cell)
+			|> Entity.give_ability(:eat, Cell)
```

Comment:
why do you need give current module here directly?
btw. probably you can use `__MODULE__`.